components/bio.js

```diff
@@ -22,10 +22,10 @@ export default function Bio({ popup = true, spanTwo = false, ...props }) {
           maxWidth: '600px',
           zIndex: !popup ? 1003 : 5,
           maxHeight: '90vh',
-          overflowY: 'scroll',
+          overflowY: 'hidden',
```

Comment:
Just curious, what necessitated this change?

Reply:
Whilst connected to an external monitor & in clamshell mode, (I'm guessing there are other cases, too), scrollbars are shown on browsers running on MacOS (not sure about Windows). The Card component has an inner scrollbar, so this hides that!
Here's a before & after!
![SCR-20230926-peiy](https://github.com/hackclub/site/assets/77097223/a2dfe1fb-3ce0-4d9b-b026-6c40ba77c0ee)
![image](https://github.com/hackclub/site/assets/77097223/1bbea844-e7d8-45dd-8eeb-f7ac3498f390)